src/components/view/ContactIntake/index.js

```diff
@@ -76,17 +73,16 @@ class IntakeForm extends Component {
             contactCountryOfBirth: '',
             contactAgeOfFirstInjection: 0,
 
-            // outreach
+            // visit or outreach
+            uid: params.uid,
+            outreach: false,
```

Comment:
on second thought, can we make this variable name `isOutreach` or something that indicates it's a boolean? kinda petty, but any change to make it a little more indicative of it's type could be useful down the line for anyone maintaining this code